WebHostLib/templates/multitracker__HK.html

```diff
@@ -0,0 +1,478 @@
+    
+    "Stag"                              : "https://cdn.wikimg.net/en/hkwiki/images/thumb/5/5e/Map_Pin_Stag.png/50px-Map_Pin_Stag.png",
+
+    "IsCompleted"                       : "https://cdn.wikimg.net/en/hkwiki/images/0/06/The_Knight_Front.png"
+} %}
+
+{% set multi_items = [
+    "Progressive Cloak",
+    "Progressive Claw",
+    "Progressive Crystal Heart",
+    "Progressive Dreamnail",
+    "Progressive Fireball",
+    "Progressive Dive",
+    "Progressive Shriek",
+    "Charm_Notch",
+    "Grimmkin_Flame",
+    "Grub",
+    "Rancid_Egg",
+    "Simple_Key",
+    "Pale_Ore",
+    "Mask_Shard",
+    "Vessel_Fragment",
+    "Essence",
+    "Stag"
+] %}
+
+{% set inventory_order = [
+    "Nail", "Monomon", "Herrah", "Lurien",
+    "Progressive Cloak", "Progressive Claw", "Progressive Crystal Heart",
+    "Monarch_Wings", "Isma's_Tear", "Progressive Dreamnail", "Dream_Gate", "Cyclone_Slash",
+    "Dash_Slash", "Great_Slash", "Progressive Fireball", "Progressive Dive", "Progressive Shriek",
+    "Focus", "Swim", "Grimmkin_Flame", "Grub", "Rancid_Egg",
+    "City_Crest", "Elegant_Key", "King's_Brand", "Love_Key", "Lumafly_Lantern", "Shopkeeper's_Key",
+    "Simple_Key", "Tram_Pass", "Elevator_Pass",
+    "Pale_Ore", "Mask_Shard", "Vessel_Fragment", "Essence", "Stag",
+    "IsCompleted"
+] %}
+
+{% set charms = [
+    "Wayward_Compass", "Gathering_Swarm", "Stalwart_Shell", "Soul_Catcher", "Shaman_Stone",
+    "Soul_Eater", "Dashmaster", "Sprintmaster", "Grubsong", "Grubberfly's_Elegy",
+    "Progressive Heart", "Progressive Greed", "Progressive Strength", "Spell_Twister", "Steady_Body",
+    "Heavy_Blow", "Quick_Slash", "Longnail", "Mark_of_Pride", "Fury_of_the_Fallen",
+    "Thorns_of_Agony", "Baldur_Shell", "Flukenest", "Defender's_Crest", "Glowing_Womb",
+    "Quick_Focus", "Deep_Focus", "Lifeblood_Heart", "Lifeblood_Core", "Joni's_Blessing",
+    "Hiveblood", "Spore_Shroom", "Sharp_Shadow", "Shape_of_Unn", "Nailmaster's_Glory",
+    "Weaversong", "Dream_Wielder", "Dreamshield", "Progressive Grimmchild", "Progressive Void Heart"
+] %}
+
+{%  set nails = {
+        "Downslash"     : { "text" : "Down" , "bit" : 1},
+        "Leftslash"     : { "text" : "Left" , "bit" : 2},
+        "Upslash"       : { "text" : "Up"   , "bit" : 4},
+        "Rightslash"    : { "text" : "Right", "bit" : 8}
+} %}
+
+{# Most have a duplicated 0th entry for when we have none of that item to still load the correct icon/name. #}
+{% set progressive_order = {
+    "Progressive Heart"             : ["Fragile_Heart", "Fragile_Heart", "Unbreakable_Heart"],
+    "Progressive Greed"             : ["Fragile_Greed", "Fragile_Greed", "Unbreakable_Greed"],
+    "Progressive Strength"          : ["Fragile_Strength", "Fragile_Strength", "Unbreakable_Strength"],
+    "Progressive Grimmchild"        : ["Grimmchild1", "Grimmchild1", "Grimmchild2", "Grimmchild3", "Grimmchild4"],
```

Comment:
grimmchild doesn't work as a progressive item in AP
the difference between grimmchild1 and grimmchild2 is 2 tells the game that it should start upgraded so the player doesn't need the grimkin flames, where 1 tells the game to start in 1 form and force the player to upgrade it with the flames in the multiworld

we could still use the different icons for whichever is in the itempool / is sent, but they are not progressive

Reply:
I would keep a single icon, barring some plando nonsense there will only ever be 1 grimmchild in the seed

Reply:
Sometimes there is Grimmchild 1 and sometimes 2. I could rewrite it in code to just Grimmchild. It is guaranteed that there is just on Grimmchild at a time?

Reply:
The progressive is there more for the visual icon. So when Grimmchild1 is in pool then we get that icon, if it's two we get the other. Progression is here more because of using the existing stuff as workaround, independend of progressive behaviour or not in game.

Reply:
Plando can cause both Grimmchilds to exist. But that can also cause, say, twenty Dream Nail items to exist or 5 version of a progressive item that only has two stages. I believe outside of plando there is only ever one Grimmchild

Reply:
https://github.com/ArchipelagoMW/Archipelago/blob/main/worlds/hk/__init__.py#L305
yes, either 1 or 2 will be added to the itempool, never both (without things like plando that we shouldn't need to cover for)
regardless it sounds like you understand they don't actually act like progression items, so whichever icons you chose to display is fine by me